man/segmentBins.Rd

```diff
@@ -15,8 +15,8 @@
 \title{Segments normalized copy number data}
 
 \usage{
-segmentBins(object, smoothBy=FALSE, alpha=0.0000000001, undo.splits="sdundo", undo.SD=1,
-  force=FALSE, transformFun="log2", mc.cores=getOption("mc.cores", 2L), ...)
+segmentBins(object, smoothBy=FALSE, alpha=1e-10, undo.splits="sdundo", undo.SD=1,
```

Comment:
Most likely differences in deparse() behavior. Maybe platform dependent, maybe R version dependent. I ran everything on Windows and R devel.